src/components/LabelGroup/component.tsx

```diff
@@ -16,13 +16,17 @@ class LabelGroup extends BaseComponent <LabelGroupArgs, any> {
         if (Array.isArray(this.props.children) || !this.props.children) {
             throw new Error('A LabelGroup must contain a single child react component');
         }
+        const labelFieldChild = (this.props.children as { type: any, props: any });
```

Comment:
Not easy to provide types here?

Reply:
I've updated the types and included an explanation.